docs/source/tutorials/cmip6-ecs.ipynb

```diff
@@ -0,0 +1,1893 @@
+{
```

Comment:
<p>It might be useful to filter these warnings and ignore them. </p><p> </p><p><code>import warnings</code></p><p><code>warnings.filterwarnings('ignore', '.*unable to decode time axis.*',)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds-funnel/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I made that change, and enabled to dataset-based cacheing! Feel free to leave comments when you get a chance